packages/compatibility/backend/sirius-components-compatibility-emf/src/main/java/org/eclipse/sirius/components/compatibility/emf/modeloperations/CreateViewOperationHandler.java

```diff
@@ -94,11 +97,17 @@ public IStatus handle(Map<String, Object> variables) {
 
         // @formatter:off
         try {
+            var containmentKind = NodeContainmentKind.CHILD_NODE;
+            if (DescriptionPackage.eINSTANCE.getAbstractNodeMapping_BorderedNodeMappings().equals(diagramElementMapping.eContainmentFeature())) {
```

Comment:
This creates a test failure:
```
java.lang.NullPointerException: Cannot invoke "org.eclipse.sirius.diagram.description.DiagramElementMapping.eContainmentFeature()" because "diagramElementMapping" is null
	at org.eclipse.sirius.components.compatibility.emf.modeloperations.CreateViewOperationHandler.handle(CreateViewOperationHandler.java:101)
	at org.eclipse.sirius.components.compatibility.emf.compatibility.operations.CreateViewOperationHandlerTests.handleAndCheckExecution(CreateViewOperationHandlerTests.java:179)
```

Reply:
I've updated the code in question, the build is now running